RNTester/Podfile.lock

```diff
@@ -293,7 +295,7 @@ DEPENDENCIES:
   - Yoga (from `../ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://github.com/cocoapods/specs.git:
+  https://github.com/CocoaPods/Specs.git:
```

Comment:
Sadly, CocoaPods is using upper case characters for their GitHub repo and org name. In order to decrease entropy, this should be kept consistent (running `pod install` will automatically change it to what's seen here). Hopefully at some point in the future CocoaPods will fix their naming.
In either case, both variants are referring to the _same_ repo.